ci/provisioning/post_provision_config_common_functions.sh

```diff
@@ -173,14 +192,14 @@ set_local_repo() {
     version="$(lsb_release -sr)"
     version=${version%%.*}
     if [ "$repo_server" = "artifactory" ] &&
-        [[ $(echo "$COMMIT_MESSAGE" | sed -ne '/^PR-repos: */s/^[^:]*: *//p') = *daos@* ]] ||
-        [[ $(echo "$COMMIT_MESSAGE" |
-             sed -ne "/^PR-repos-$DISTRO: */s/^[^:]*: *//p") = *daos@* ]] ||
-        [ -z "$(echo "$COMMIT_MESSAGE" | sed -ne '/^RPM-test-version: */s/^[^:]*: *//p')" ]; then
-        # Disable the daos repo so that the Jenkins job repo is used for daos packages
+       [ -z "$(rpm_test_version)" ] &&
+       [[ $BRANCH_NAME != weekly-testing* ]]; then
```

Comment:
I need to double-check that this is the correct variable.  I might actually want `CHANGE_BRANCH` here, not `BRANCH_NAME`.

Reply:
Given that this only needs to check for the `weekly-testing*` branches and is orthogonal to PR branches, `BRANCH_NAME` should be sufficient here.